Raise exceptions control

avpeshkov opened the pull request:
Hi all, I've been working with pdfkit and found several cases where wkhtmltopdf outputs errors, but still generates a pdf.  I would like to be able to manually control how the pdf-kit responds to errors.

ps: I apologize if I violate protocol in any way, this is my first time doing pull request to opensource.

JazzCore commented:
Hi, thanks for the work! Not sure about it though, since when using this option any errors will be ignored, even when pdf is not generated. More correct way is to check if we got any output and if there are errors - we can ignore those, and if output is missing - properly raise an exception

avpeshkov commented:
> Hi, thanks for the work! Not sure about it though, since when using this option any errors will be ignored, even when pdf is not generated. More correct way is to check if we got any output and if there are errors - we can ignore those, and if output is missing - properly raise an exception

Hey, 
Thanks for the quick review! I have taken your remarks into account and updated pull request, also I added info about the changes to README.

avpeshkov commented:
Hi, again:)
I fixed tests to get python2 compatibility. Who would have thought there would be such a mistake in workflow:)

JazzCore commented:
Looks like GH CI is broken atm... No time to review at the moment, sorry. I'll try to test it on this weekend

avpeshkov commented:
Thank you, this errors looks vary unusual.
No problem, we all have a job:)
